riscv/insns/vfncvt_f_f_w.h

```diff
@@ -1,9 +1,7 @@
-// vfncvt.f.f.v vd, vs2, vm
+// vfncvt.f.f.w vd, vs2, vm
 VI_VFP_NCVT_FP_TO_FP(
-  {;},                                 // BODY16
```

Comment:
Deleting BODY16 suggests that the PR description is incomplete.  I guess BODY16 is removed for narrowing ops and BODY8 is removed for single-width ops?

Reply:
OK. I have udpated the description and add a new commit to update the parameters for VI_VFP_NCVT_* macros to be consistent with these comments.